tfhe/src/high_level_api/keys/public.rs

```diff
@@ -66,7 +60,7 @@ pub trait RefKeyFromPublicKeyChain: Sized {
     }
 }
 
-#[cfg(any(feature = "integer", feature = "shortint", feature = "boolean"))]
```

Comment:
Why remove integer here?

Reply:
because this macro is not used in integer part of the API